### PR TITLE
Relay: Fixed issue #8775 : 

### PR DESCRIPTION
**Pull request by v-Ajnava**
Fixed issue #8775 : Cannot create hybrid connection with disabled client authroization

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [X] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/HISTORY.rst [hunk 1]
```diff
@@ -3,6 +3,11 @@
 Release History
 ===============
 
+**Relay**
```
Comment (tjprescott) [inline src/azure-cli/HISTORY.rst]: This needs to be alphabetical in the list.

File: src/azure-cli/azure/cli/command_modules/relay/_params.py [hunk 2]
```diff
@@ -71,13 +71,15 @@ def load_arguments_sb(self, _):
...
         c.argument('requires_client_authorization', arg_type=get_three_state_flag(), help='Indicates whether client authorization is required')
+        c.argument('requires_transport_security', arg_type=get_three_state_flag(), help='Indicates whether transport security is required')
         c.argument('user_metadata', help='Endpoint metadata')
 
     with self.argument_context('relay wcfrelay update') as c:
         c.argument('relay_name', arg_type=name_type, id_part='child_name_1', help='Name of WCF Relay')
         c.argument('status', arg_type=get_enum_type(['Active', 'Disabled', 'SendDisabled', 'ReceiveDisabled']), help='Enumerates the possible values for the status of a messaging entity.')
         c.argument('relay_type', arg_type=get_enum_type(['Http', 'NetTcp']), default='NetTcp', help='Relay type')
         c.argument('requires_client_authorization', arg_type=get_three_state_flag(), help='Indicates whether client authorization is required')
+        c.argument('requires_transport_security', arg_type=get_three_state_flag(), help='Indicates whether transport security is required')
```
Comment (tjprescott) [inline src/azure-cli/azure/cli/command_modules/relay/_params.py]: You don't need this duplication. This should just be registered under a scope of 'relay wcfrelay'.

Comment (v-Ajnava) [reply src/azure-cli/azure/cli/command_modules/relay/_params.py]: Let me check with PM, as 'requires_client_authorization' and 'requires_transport_security' can be set only at create wcfrelay and can not be changed. I think its should be removed from update command.

File: src/azure-cli/HISTORY.rst (hunk 1 above)
Comment (v-Ajnava) [reply src/azure-cli/HISTORY.rst]: sur, will make change

**Pull request merged**